website/docs/patterns/table-multi-select/partials/interaction-concepts/interaction-concepts.md

```diff
@@ -0,0 +1,69 @@
+When combining a Multi-select table with filtering, sorting, or pagination, you may encounter complex scenarios regarding how selected results should be persisted across pages and what the user expects when traversing large data sets.
+
+These interaction guidelines are not fully comprehensive. They should instead be considered as conversation starters between design and engineering on how best to optimize the user experience.
+
+!!! Info
+
+If you run into unique scenarios that these guidelines don’t account for, we would love to hear about them through our normal [support](/about/support) channels.
+!!!
+
+## Consistent communication
+
+To help prevent unnecessary editing or deleting and to ensure users are interacting with the results they intend to, it is vital to consistently communicate their selections, or the existence of a selection, **at all times**.
+
+Persisting this communication in the selected count can help to avoid mishaps, but sometimes it might be necessary to confirm with the user prior to making changes that cannot be reversed.
+
+![Example of a double confirmation](/assets/patterns/table-multi-select/multi-select-double-confirmation.png)
+
+## Selection scope
+
+_Where_ a selection is made in the UI can imply differences in how the selection is scoped, and at a very basic level can broken down into three different hierarchies.
+
+1. **Global scope:** selection applies to all rows in the table and is made via the bulk selection in the Multi-select pattern.
+2. **Page-level scope:** selection applies to all rows in the current page of a paginated table and is made via the Checkbox in the Table header.
+3. **Row-level scope:** selection applies only to a single row, result, or item in a Table, and is made via the checkbox at the start of the row.
+
+![Selection scope](/assets/patterns/table-multi-select/selection-scope.png)
```

Comment:
In this image/representation the table doesn't have a pagination, so it's confusing what "page-level" scope refers to